src/Pipeline.cpp

```diff
@@ -340,7 +340,7 @@ void Pipeline::compile_to_lowered_stmt(const string &filename,
                                        StmtOutputFormat fmt,
                                        const Target &target) {
     Module m = compile_to_module(args, "", target);
-    m.compile(single_output(filename, m, fmt == HTML ? OutputFileType::stmt_html : OutputFileType::stmt));
+    m.compile(single_output(filename, m, fmt == HTML ? OutputFileType::stmt_html : (fmt == StmtViz ? OutputFileType::stmt_viz : OutputFileType::stmt)));
```

Comment:
Similarly, should we first generate the assembly file here if the target format is `stmt_viz`?